moto/cloudformation/responses.py

```diff
@@ -87,6 +93,19 @@ def _get_param_values(self, parameters_list, existing_params):
                 raise MissingParameterError(parameter["parameter_key"])
         return result
 
+    def process_cfn_response(self):
```

Comment:
This method ultimately deals with the HTTP-request made by the `cfnresponse`-module.

 - Find the stack
 - Find the custom resource that called this URL
 - Persist any outputs that are returned by the custom resource
 - Update the stack-status to mark it as `create_complete`, if applicable